src/Components/Table/uFuzzy/uFuzzy.ts

```diff
@@ -0,0 +1,45 @@
+import uFuzzy from '@leeoniya/ufuzzy';
```

Comment:
Nit, but the module is leaking implementation details 😄 . Maybe just `search.ts`.